Sources/SwiftInspectorVisitors/TypeSyntaxVisitor.swift

```diff
@@ -233,51 +248,12 @@ public struct TypeProperties: Hashable {
     }
   }
 
-  public struct PropertyData: Hashable {
-    /// The name of the property
-    public let name: String
-    /// The Type annotation of the property if it's present
-    public let typeAnnotation: String?
-    /// Any comments associated with the property
-    public let comment: String
-    /// Modifier set for this type
-    public let modifiers: Modifier
-  }
-
-  /// The name of the type.
+  /// The name of the property
   public let name: String
-  /// The properties on this type
-  public let properties: Set<PropertyData>
-}
-
-extension TypeProperties {
-  struct MergeError: Error, CustomStringConvertible {
```

Comment:
Assuming we have good test coverage that we don't surface properties for types that we're not looking for, I don't think we need this.